cmd/katalyst-agent/app/options/orm/orm_base.go

```diff
@@ -58,6 +64,12 @@ func (o *GenericORMPluginOptions) AddFlags(fss *cliflag.NamedFlagSets) {
 		o.TopologyPolicyName, "topology merge policy name used by ORM")
 	fs.StringSliceVar(&o.NumericAlignResources, "numeric-align-resources", o.NumericAlignResources,
 		"resources which should be aligned in numeric topology policy")
+	fs.StringVar(&o.ORMPodResourcesSocket, "orm-pod-resources-socket", o.ORMPodResourcesSocket,
+		"socket of ORM pod resource api, default 'unix:/var/lib/katalyst/pod-resources/kubelet.sock'")
+	fs.StringVar(&o.ORMDevicesProvider, "orm-devices-provider", o.ORMDevicesProvider,
```

Comment:
If this config is empty, does it mean that ORM has no requirements for the K8s version?

Reply:
> If the `orm-devices-provider` config is empty, does it mean that ORM has no requirements for the K8s version?

Yes,  there is no other rpc or http connections between ORM and kubelet.